app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__MarkdownBlockEditor.res

```diff
@@ -12,5 +12,4 @@ let make = (~markdown, ~contentBlock, ~updateContentBlockCB) =>
     profile=Markdown.Permissive
     maxLength=10000
     onChange={onChange(contentBlock, updateContentBlockCB)}
-    dynamicHeight=true
```

Comment:
@indicozy - This change is unrelated to this PR. Could you please stash or move them to a new PR?